example_test.go

```diff
@@ -998,6 +998,90 @@ func ExampleBeginRequest_TxnIsolation() {
 	fmt.Printf("Select after Rollback: response is %#v\n", data)
 }
 
+func ExampleBeginRequest_IsSync() {
+	conn := exampleConnect(dialer, opts)
+	defer conn.Close()
+
+	// Tarantool supports IS_SYNC flag for BeginRequest since version 3.1.0
```

Comment:
A comment should end with a dot.
```suggestion
	// Tarantool supports IS_SYNC flag for BeginRequest since version 3.1.0.
```